assessment_software/src/app/app.component.html

```diff
@@ -0,0 +1,3 @@
+<app-textfield placeholder="email" 
```

Comment:
The type has not been set. Without the type, it will default to text field that accepts texts only.

Reply:
#12 @Aben-Bel type is set to "email"

Reply:
@Aben-Bel